content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,20 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/apply-time-mutation
+
+Type: Annotation
+
+Example: `config.kubernetes.io/apply-time-mutation: "pods"`
```

Comment:
Try this:
```suggestion
Example:
```yaml
    config.kubernetes.io/apply-time-mutation: |
      - sourceRef:
          kind: Pod
          name: pod-a
        sourcePath: $.status.podIP
        targetPath: $.spec.containers[?(@.name=="example")].env[?(@.name=="SERVICE_HOST")].value
        token: ${pod-a-ip}
```

I think that's a good start